week4project/src/test/java/xpathProject/xPath.java

```diff
@@ -0,0 +1,111 @@
+package xpathProject;
```

Comment:
I only needed the actual selectors. Like the following (these are also the solutions)

1. Page: /text-box. Write the xpath that will return the ‘Full Name’textbox using the ID attribute. Hint: This element has the ‘input’tag.

//input[@id="userName"]

2. Page: /buttons. Write the xpath that will return all elements on the page with the ‘button’tag.

//button

3. Page: /webtables. Write the xpath that will return the div with text of ‘Vega’. Ensure to narrow the scope of your xpath to the ReactTable and 
ensure that your xpath is limited to returning cells only. Cells in this table have the class ‘rt-td’. Hint: You must use “Vega” in your xpath.

//div[@class='ReactTable -striped -highlight']//div[text()='Vega']

4. Page: /radio-button.Write the xpath that will return the parent element for the radio button with id=’impressiveRadio’.

//input[@id="impressiveRadio"]/div::parent
or
//input[@id="impressiveRadio"]/div/..

5. Page: /radio-button. Write the xpath that will return all DIV ancestors for the radio button with id=’impressiveRadio’.

//input[@id="impressiveRadio"]/ancestor::div

6.. Page: /radio-button: Write the xpath that will return the sibling for the radio button with id=’impressiveRadio’. Do not use the sibling axes. 

//input[@id="impressiveRadio"]/div/..//label

7. Page: /buttons. Write the xpath that will return all buttons containing the value “Click”.

//button[contains(text(),"Click")]



CSS Project
1. Page: /text-box. Write a selector that will return the ‘Full Name’ textboxusing the ID attribute. Hint: This element has the ‘input’ tag.

input[id="userName"]

2. Page: /buttons. Write a selector that will return all 3 buttons shown below.

button

3. Page:/buttons. Write a selector that will return the span element in the footer.
 
footer>span

4. Page:/menu.Write a selector that will return the 3 listitem elements in the unordered list.

ul[id="nav"]>li

5. Page: /select-menu. Write a selector that will get the option element where value = 4.

div#selectMenuContainer option[value="4"]

6. Page: /slider. Write the JavaScript to get the value for the element with id=sliderValue. 

document.getElementsByClassName('range-slider range-slider--primary')[0].value

7. Page: /text-box. Write the JavaScript to seet the value for the element with id=username.

document.getElementsByClassName('range-slider range-slider--primary')[0].value=30